### DISCUSSION TITLE
my copilot can not access

### DISCUSSION THREAD

**Discussion by esrrhs** [community/community#116648] [Copilot Conversations]
### Select Topic Area

Question

### Body

Hi, my [Billing Add-ons](https://github.com/settings/billing/summary) shows ```You are eligible to use GitHub Copilot for free.``` And I use copilot for months
but I could not use the copilot since a few day ago, it ask me to login, and then says ```Contact Support. You are currently logged in as esrrhs.```
and the [Copilot](https://github.com/settings/copilot) page shows ```Buy Copilot```, but I click the button, nothing happened.
Can anyone help me? I created a [ticket](https://support.github.com/ticket/personal/0/2698834) last week, but no response.
maybe my account was blocked? but I just normal use.

Thanks.

**Comment by jiangyijie27** [comment]
same problem
